pkg/triggerconfig/inrepo/test_data/load_pipelinerun/pipeline-finally-params/README.md

```diff
@@ -0,0 +1,3 @@
+## Reuse steps in a task inherit a 'finally' block
+
+This example shows how we can use `image: uses:sourceURI` and a `name: mystep` to include individual the steps in task 
```

Comment:
I guess this README should be about the params and not the `image: uses:sourceURI` which should be on another PR as you said

Reply:
done